dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -116,6 +116,7 @@ generate_configuration_with_puppet() {
     # Handle override/addon
     if [ -d "/repo/dockerfiles/init/addon" ]; then
       WRITE_PARAMETERS+=" -v \"${CHE_HOST_DEVELOPMENT_REPO}/dockerfiles/init/addon/addon.pp\":/etc/puppet/manifests/addon.pp:ro"
+      WRITE_PARAMETERS+=" -v \"${CHE_HOST_DEVELOPMENT_REPO}/dockerfiles/init/addon/modules/\":/etc/puppet/addon/:ro"
```

Comment:
no it can't be sub folder, puppet will not be able to find it. in module path puppet expects scructure where `/etc/puppet/modules/<module_name>/{manifests},{modules}` but we will have `/etc/puppet/modules/addon/<module_name>/{manifests},{modules}` I've tried this.